src/main/scala/firrtl/util/Utils.scala

```diff
@@ -1,16 +1,15 @@
 // See LICENSE for license details.
```

Comment:
In moving this to a Utils package, I think we should also rename `object Utils` -> `package object util` so that we don't have to write things like `firrtl.util.Utils.throwInternalError`, instead writing `firrtl.util.throwInternalError`.